unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Audio/DCLAudioClip.cs

```diff
@@ -135,7 +132,7 @@ public override IEnumerator ApplyChanges(BaseModel newModel)
         public override void Dispose()
         {
             isDisposed = true;
-            Utils.SafeDestroy(audioClip);
+            AssetPromiseKeeper_AudioClip.i.Forget(audioClipPromise);
```

Comment:
Should we unsuscribe there from the promise? 
```

  audioClipPromise.OnSuccessEvent -= OnComplete;
                audioClipPromise.OnFailEvent -= OnFail;
```

Reply:
after failing, succeeding or forgotten promises clear those events, so there is no need to unsubscribe them